nautobot/extras/tests/test_changelog.py

```diff
@@ -23,17 +23,16 @@ class ChangeLogViewTest(ModelViewTestCase):
 
     @classmethod
     def setUpTestData(cls):
-
         # Create a custom field on the Location model
         ct = ContentType.objects.get_for_model(Location)
-        cf = CustomField(type=CustomFieldTypeChoices.TYPE_TEXT, name="my_field", required=False)
+        cf = CustomField(type=CustomFieldTypeChoices.TYPE_TEXT, label="my_field", required=False)
         cf.validated_save()
         cf.content_types.set([ct])
 
         # Create a select custom field on the Location model
         cf_select = CustomField(
             type=CustomFieldTypeChoices.TYPE_SELECT,
-            name="my_field_select",
+            label="my_field_select",
```

Comment:
Nit: let's use human-readable labels for these, i.e. `My Text Field` and `My Select Field`